CONTRIBUTING.md

```diff
@@ -330,6 +330,18 @@ Use your real name (sorry, no pseudonyms or anonymous contributions).
 
 If you set your `user.name` and `user.email` git configs, you can sign your commit automatically with `git commit -s`.
 
+#### Automatic DCO Signing (Recommended)
+
+To automatically sign all your commits and avoid DCO check failures, configure Git to use the repository's hooks:
+
+```shell
+$ make init-git-hooks
+```
+
+This will configure Git to automatically add the `Signed-off-by` trailer to all your commit messages.
+This is the recommended approach as it eliminates the need to remember the `-s` flag and prevents DCO-related CI failures.
```

Comment:
Personally I am not sure I would call this recommend, I at least would still recommend using -s as it serves as an explicit reminder that yes I do in fact have the right to submit this.

The alias thing mentioned @Honny1 seem more more useful to me. That doesn't require us to add this some hook logic to basically any repo we maintain for contributors to use. Because I rather not have this duplicated anywhere.